src/components/Navbar/constants.js

```diff
@@ -0,0 +1,3 @@
+export const navbarConstants = {
+  height: '64px',
+};
```

Comment:
Why are we using these component?

Reply:
Look where is use and think about it

Reply:
Let me know if you find a reason to use it

Reply:
you use it in the navbar.styles.js . 
 Is there a reason why you choose not to set the height directly in the navbar.styles.js component ?

Reply:
Things that doesn't change should be constants. We could need it that variable in another field.

Reply:
> Why are we using these component?

When you have question like this, you should specify a little more what you want to know, give me some context. I don't know if you think we shouldn't use it or if you don't know why we are using it, those tow are different things and it would be good if I know exactly what you want to know.

Reply:
Thanks for your Feedback, I will make it clearer next time.
With this question i meant it as i said: why are we using it, cause i did not see these constant useful somewhere else.